managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedCursorImpl.java

```diff
@@ -1823,8 +1825,10 @@ void internalFlushPendingMarkDeletes() {
 
     void createNewMetadataLedger(final VoidCallback callback) {
         ledger.mbean.startCursorLedgerCreateOp();
+        final long now = System.currentTimeMillis();
```

Comment:
Creating a BK ledger involves 3 ZK write operations. Either we account for that or we treat it separately